docker-compose.yml

```diff
@@ -147,8 +147,8 @@ services:
       - POSTGRES_USER=${POSTGRES_DB_USER:-postgres}
       - POSTGRES_PASSWORD=${POSTGRES_DB_PASSWD:-password}
     healthcheck:
-      test: ["CMD", "pg_isready"]
-      interval: 10s
```

Comment:
Add comment on why this needs to be 15s (see your pull request description)

Reply:
I did some further investigation to find an explanation to why this is happening. It turns out the healthcheck command "pg_isready" only checks if postgres is ready to accept connections. When I look at the log I see the boundary_service trying to connect after 10 seconds, while the postgres database is still being setup.

The postgres container reports healthy because it is ready to accept connections, but at that time the database is still being prepared and I believe therefore the boundary_service container can't connect to it, causing the error. I am not 100% sure on this.

Changing the interval to 15s fixes this error, because I can see that the database is properly setup by this time. There is a slight issue with this. The time it takes for the database to be fully configured, differs with hardware. 

I will include this information in the comment.

Reply:
Alright awesome.